### PR TITLE
Add receipes for Enqueue 0.9.

### PR DESCRIPTION
**Pull request by makasim**
| Q             | A
| ------------- | ---
| License       | MIT

fixes https://github.com/php-enqueue/enqueue-dev/issues/600

### CODE REVIEW

**Comment by makasim** [comment]
@Nyholm not yet

File: enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+enqueue:
+    default:
+        transport: 'null:'
```
Comment (OskarStark) [inline enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml]: `null:` or `null://`

File: enqueue/wamp/0.9/manifest.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "wamp:"
```
Comment (OskarStark) [inline enqueue/wamp/0.9/manifest.json]: `wamp:` or `wamp://` ?

**Comment by OskarStark** [comment]
Can someone please rerun the travis job?

File: enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml (hunk 1 above)
Comment (makasim) [reply enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml]: `null:` is a valid scheme.

Comment (OskarStark) [reply enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml]: I know, but then we should use it here too:
https://github.com/symfony/recipes-contrib/pull/565/files#diff-d1033b7e1d7ea30fd6e9eae6a48948e8R9

I am against mixing it with or without `//`

**Comment by Nyholm** [comment]
I reran travis. I know that we recently made an update to support multiple packages in one PR. A rebase will fix that.

**Comment by Nyholm** [comment]
When I test this I get: 

>  A given scheme "wamp" is not supported. Maybe it is a custom driver, make sure you registered it with "Enqueue\Client\Resources::addDriver".

Is that expected? If I removed the second `ENQUEUE_DSN` from my `.env` it all works great.

**Comment by OskarStark** [comment]
Yes I will fix that in another PR, we only want to expose the ENQUEUE_ENV with the bundle and not for each transport... 

see https://github.com/symfony/recipes-contrib/pull/568#issuecomment-442729478 and the following comments

**Pull request merged**